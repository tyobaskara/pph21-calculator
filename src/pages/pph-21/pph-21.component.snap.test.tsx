import React from "react";
import renderer from 'react-test-renderer';
import PPH21Page from './pph-21.component';

jest
  .mock('../../components', () => ({
    Navigation: 'NavigationComponent',
    FormPPH21Page: 'FormPPH21PageComponent'
  }));

test('PPH21Page snaptest', () => {
  const component = renderer.create(
    <PPH21Page/>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});