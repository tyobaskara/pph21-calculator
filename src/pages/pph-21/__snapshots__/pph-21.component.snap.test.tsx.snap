// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PPH21Page snaptest 1`] = `
<div>
  <NavigationComponent
    path="/"
  />
  <main>
    <div
      className="container"
    >
      <FormPPH21PageComponent />
    </div>
  </main>
</div>
`;
